typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -81,23 +81,14 @@ export abstract class HyperlaneAppChecker<
 
   async checkProxiedContracts(
     chain: ChainName,
-    proxyAdminAddress?: Address,
+    owner: Address,
+    ownableOverrides?: Record<string, Address>,
   ): Promise<void> {
     const expectedProxyAdminAddress =
-      proxyAdminAddress ?? this.app.getContracts(chain).proxyAdmin.address;
-    if (!expectedProxyAdminAddress) {
```

Comment:
The `expectedProxyAdminAddress` is no longer required. This is to support the case where we don't own the proxy and therefore will not know the proxyAdmin address before deployment i.e Renzo warp routes

if `expectedProxyAdminAddress` is not set, we will not check for proxyAdmin address violations. 

For AW owned proxies, we will check that the `proxyAdmin` is the singleton `proxyAdmin`